core/src/main/scala/kafka/server/AclApis.scala

```diff
@@ -69,7 +69,7 @@ class AclApis(authHelper: AuthHelper,
       case Some(auth) =>
         val filter = describeAclsRequest.filter
         val returnedAcls = new util.HashSet[AclBinding]()
-        auth.acls(filter).forEach(returnedAcls.add)
+        returnedAcls.add(auth.acls(filter).iterator().next())
```

Comment:
Originally we will add all `auth.acls(filter)` content into `returnedAcls`, but after this change, we only add the first element. Why do we need this change?

Reply:
OK, I know why you did this change now.
> we don't need to collect all items to a new collection, right?

I believe what @chia7712 meant is we don't have to do a collection copy here because we already allow passing `Iterable` into `aclsResource` method. To make it clear, in the JIRA, we said:
> 1. Iterable -> HashSet

This is the collection copy we want to avoid here. So we don't need `returnedAcls` anymore, we can pass the result of `auth.acls(filter)` into `aclsResource` directly. Is that clear?

Reply:
Hi, @showuon 
Thanks for the explanation. It's crystal clear. Would apply.